config/module.go

```diff
@@ -248,6 +249,7 @@ func (m Module) EvaluateFirstRunPath(packagesDir string) (
 
 	firstRunSuccessPath := unpackedModDir + FirstRunSuccessSuffix
 	if _, err := os.Stat(firstRunSuccessPath); !errors.Is(err, os.ErrNotExist) {
+		logger.Info("first run already ran")
```

Comment:
will this not be double logged?

Reply:
it shouldn't - where do you see potential for a double log?

Reply:
because it returns an error saying the same thing - just wondering, wasn't sure if that would happen

Reply:
oh yeah, sorry you are right - the error  from this function is logged at the debug level, so we get a double log in that scenario. this should get cleared up when we refactor this function. the current contract requires we return an error here to prevent first run logic from firing. we can change the text in the error to prevent a double-log in the meantime.

Reply:
since it only happens during debug, I think it's better to address during the upcoming refactor PR